python/benchmarks/bench_utils.py

```diff
@@ -39,7 +39,7 @@
 
 def read_file(uri) -> bytes:
     if not urlparse(uri).scheme:
```

Comment:
are you allowed to use `~` substitution in `file://` uri's?

Reply:
Yea, this way we can use "train.py ~/dataset".